website/docs/quick-start-guide.md

```diff
@@ -246,67 +246,86 @@ Spark SQL needs an explicit create table command.
 
 **Table Concepts**
 
-- Table types
+- **Table types**
 
   Both Hudi's table types, Copy-On-Write (COW) and Merge-On-Read (MOR), can be created using Spark SQL.
   While creating the table, table type can be specified using **type** option: **type = 'cow'** or **type = 'mor'**.
 
-- Partitioned & Non-Partitioned tables
+- **Partitioned & Non-Partitioned tables**
 
   Users can create a partitioned table or a non-partitioned table in Spark SQL. To create a partitioned table, one needs
   to use **partitioned by** statement to specify the partition columns to create a partitioned table. When there is
   no **partitioned by** statement with create table command, table is considered to be a non-partitioned table.
 
-- Managed & External tables
+- **Primary keyed table**
 
-  In general, Spark SQL supports two kinds of tables, namely managed and external. If one specifies a location using **
-  location** statement or use `create external table` to create table explicitly, it is an external table, else its
-  considered a managed table. You can read more about external vs managed
-  tables [here](https://sparkbyexamples.com/apache-hive/difference-between-hive-internal-tables-and-external-tables/).
+  Optionally users can choose to create a Primary keyed table. When primary key is set for a given table, 
+  Hudi ensures uniqueness during updates and deletes. Each record is uniquely identified by the primary key configuration. 
+  If primary key is not set, Hudi treats it as key less table and every record ingested is treated as a new record even 
+  if contents match. 
 
-*Read more in the [table management](/docs/table_management) guide.*
+:::note
+1. Since Hudi 0.14.0, users can create key less table or primary keyed table as per necessity. If 'primaryKey' 
```

Comment:
Is this keyless option available for tables created via DeltaStreamer or spark datasource as well?

Reply:
yes, we do have.